src/test/isolation2/input/resgroup/resgroup_oom.source

```diff
@@ -0,0 +1,48 @@
+-- start_ignore
+CREATE RESOURCE GROUP rg_oom_test
+WITH (cpu_rate_limit=20, memory_limit=20, memory_shared_quota=100);
+CREATE ROLE role_oom_test RESOURCE GROUP rg_oom_test;
+
+CREATE EXTENSION gp_inject_fault;
+
+CREATE OR REPLACE FUNCTION gp_mock_cdbdispatchcommand(amount int)
+RETURNS SETOF text AS '@abs_srcdir@/../regress/regress.so',
+'gp_mock_cdbdispatchcommand' LANGUAGE C;
```

Comment:
I think you should remove the group, role and function in the ignore block, and create them elsewhere.